cmd/livepeer/livepeer.go

```diff
@@ -202,6 +203,7 @@ func main() {
 		},
 		"mainnet": {
 			ethController: "0xf96d54e490317c557a967abfa5d6e33006be69b3",
+			minGasPrice:   big.NewInt(1e9),
```

Comment:
Should we use [params.GWei](https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/params/denomination.go#L26)?

Reply:
It seems like the compiler looks at `int64(params.GWei)` as a constant still rather than a variable that needs to convert the untyped constant to an int64 at runtime so I'm okay with this.